en/docs/chapter_graph/graph.md

```diff
@@ -19,18 +19,18 @@ If vertices are viewed as nodes and edges as references (pointers) connecting th
 Based on whether edges have direction, graphs can be divided into <u>undirected graphs</u> and <u>directed graphs</u>, as shown in the figure below.
 
 - In undirected graphs, edges represent a "bidirectional" connection between two vertices, for example, the "friendship" in WeChat or QQ.
-- In directed graphs, edges have directionality, that is, the edges $A \rightarrow B$ and $A \leftarrow B$ are independent of each other, for example, the "follow" and "be followed" relationship on Weibo or TikTok.
+- In directed graphs, edges have directionality, that is, the edges $A \rightarrow B$ and $A \leftarrow B$ are independent of each other, for example, the "follow" and "followed" relationship on Instagram or TikTok.
 
 ![Directed and undirected graphs](graph.assets/directed_graph.png)
 
 Based on whether all vertices are connected, graphs can be divided into <u>connected graphs</u> and <u>disconnected graphs</u>, as shown in the figure below.
 
-- For connected graphs, it is possible to reach any other vertex starting from a certain vertex.
-- For disconnected graphs, there is at least one vertex that cannot be reached from a certain starting vertex.
+- For connected graphs, it is possible to reach any other vertex starting from an arbitrary vertex.
+- For disconnected graphs, there is at least one vertex that cannot be reached from an arbitrary starting vertex.
 
 ![Connected and disconnected graphs](graph.assets/connected_graph.png)
 
-We can also add a weight variable to edges, resulting in <u>weighted graphs</u> as shown in the figure below. For example, in mobile games like "Honor of Kings", the system calculates the "closeness" between players based on shared gaming time, and this closeness network can be represented with a weighted graph.
+We can also add a weight variable to edges, resulting in <u>weighted graphs</u> as shown in the figure below. For example, in Instagram, the system sorts your follower and following list by the level of interaction between you and other users (likes, views, comments, etc.), and this interaction network can be represented with a weighted graph.
```

Comment:
It's better to put the last piece of sentence into a separate one. "Such an interaction network can be represented by a weighted graph."

Reply:
Taken